api/README.rst

```diff
@@ -1,86 +1,71 @@
-===================
-Crate Benchmark API
-===================
-
-Installation
-============
-
-The `Flask`_ application is bootstrapped using virtualenv::
+=============
+Benchmark API
+=============
 
-  python3.4 -m venv env
-  source env/bin/activate
-  pip install --upgrade pip
-  pip install -e .
+A Flask_ application that reads CrateDB benchmark results from Microsoft Azure.
 
-Usage
+Setup
 =====
 
-There are 2 commands: ``server`` and ``add-timestamp``::
+Bootstrap the application like so::
 
-  > bench-api --help
-  usage: bench-api [-h] --conf CONF {server,add-timestamp} ...
+    $ python3.4 -m venv env
+    $ source env/bin/activate
+    $ pip install --upgrade pip
+    $ pip install -e .
 
-  positional arguments:
-    {server,add-timestamp}
-
-  optional arguments:
-    -h, --help            show this help message and exit
-    --conf CONF           Path to config.tomUsage
-
-server
-------
-
-Run API that reads benchmark result data from Crate cluster::
+Usage
+=====
 
-  > bench-api --conf config.toml server
+The ``bench-api`` CLI tool has two commands: ``server`` and ``add-timestamp``.
 
-For help::
+To read data from the CrateDB cluster, use the ``server`` command::
 
-  > bench-api server --help
+    $ bench-api --conf config.toml server
 
+For help, run::
 
-add-timestamp
--------------
+    $ bench-api server --help
 
-Resolve timestamp of Github commit hash and store it in doc.benchmarks table::
+To resolve the timestamp of a GitHub commit hash and store it in
+``doc.benchmarks`` table, run::
 
-  > bench-api --conf config.toml add-timestamp
+    $ bench-api --conf config.toml add-timestamp
 
-For help::
+For help, run::
 
-  > bench-api add-timestamp --help
+    $ bench-api add-timestamp --help
 
 REST Endpoint
 =============
 
-::
+This app exposes the following REST endpoint:
 
-  -> GET /result/<benchmark-group>?[&from=<isotime>&to=<isotime>]
-  [{...}, {...}, ...] # Crate data format
+    /result/<BENCHMARK_GROUP>?[&from=<FROM_ISOTIME>&to=<TO_ISOTIME>]
 
-The API call will result in this query:
+This results in the following query:
 
 .. code-block:: sql
 
-  SELECT ? as "group",
-    version_info['number'] as "version",
-    version_info['timestamp'] as "version",
-    runtime_stats['min'] as "min",
-    runtime_stats['median'] as "median",
-    runtime_stats['max'] as "max",
-    runtime_stats['stdev'] as "stdev",
-    runtime_stats['variance'] as "variance",
-    statement
-  FROM "benchmark"."history"
-  WHERE statement = ANY(?)
-    AND version_info['build_timestamp'] >= ?
-    AND version_info['build_timestamp'] <= ?
-  ORDER BY version, statement
+      SELECT @BENCHMARK_GROUP as "group",
```

Comment:
nope. but I wanted to make it clear to the reader which values get substituted with what. I originally wrote them as <BENCHMARK_GROUP> but then that messes up the SQL syntax highlighting. so I switched to using SQL variable syntax

I forgot to add a para explaining this though. would you be happy if I added one saying, "here, `$BENCHMARK_GROUP` is substituted for `<BENCHMARK_GROUP>` in the API call" or something similar?